var/spack/repos/builtin/packages/magma/package.py

```diff
@@ -56,7 +55,8 @@ class Magma(CMakePackage, CudaPackage):
     patch('magma-2.5.0.patch', when='@2.5.0')
     patch('magma-2.5.0-cmake.patch', when='@2.5.0')
     patch('cmake-W.patch', when='@2.5.0:%nvhpc')
-    patch('sm_37.patch', when='@2.6.0 cuda_arch=37')
+    # Disabled patch of v2.6.0 until version is officially released
+    #patch('sm_37.patch', when='@2.6.0 cuda_arch=37')
```

Comment:
If this patch will be included in 2.6.0, then you can just remove this and the patch file too.

Reply:
Yes, if it is included in the release, then I"ll plan to remove the patch. I can't say for sure if it will be included though.